### PR TITLE
Collapse top menu navigation at smaller screen resolution

### LINKED ISSUE #44: Top menu disappears when small.
localhost_5000_dev_html__players

When the window's width is small, the navigation disappears.

### PR DESCRIPTION
**Pull request by akagomez**
- Add `bootstrap/js/collapse` as a dependency
- Configure `bootstrap/js/collapse` dependency
- Add collapse menu icon to `navigation.stache` markup

The resulting top menu at smaller display resolutions: 

screen shot 2016-02-22 at 2 14 19 pm

Closes #44.

### CODE REVIEW

**Comment by justinbmeyer** [comment]
@akagomez what does the menu show when collapsed?  Is it possible to keep the tournaments and players links?

**Comment by justinbmeyer** [comment]
To test this will be tricky.  The best way might be to use FuncUnit to open an iframe a particular size and then verify these buttons are visible.

**Comment by akagomez** [comment]
Collapsed: 



Expanded:



I'll look into avoiding the collapsing all together.

**Comment by akagomez** [comment]
Before I write tests for this, should I first see about avoiding the collapsing all together and maintaining the existing layout?

**Comment by justinbmeyer** [comment]
@akagomez yes if it should be able to fit in that space.

What does Login look like expanded in what you currently made?

**Comment by akagomez** [comment]
No different than it does currently:

**Comment by justinbmeyer** [comment]
I mean't when the window's width is small.  What does the following look like expanded:


collapse_top_menu_navigation_at_smaller_screen_resolution_by_akagomez_ _pull_request__62_ _donejs_bitballs
.

**Comment by akagomez** [comment]
I was able to prevent the collapsing all together and maintain the existing navigation layout at smaller screen resolutions by overriding the mobile-first defaults that Bootstrap defines. 

A more elegant way to have achieved this may have been to set `@grid-float-breakpoint` in Bootstrap's Less, but we aren't building Bootstrap from source. 

The current/latest small screen resolution layout: 



And the larger screen resolution layout: 



@justinbmeyer Would you like me to write a test against this revision?

**Comment by justinbmeyer** [comment]
Yes

File: public/components/navigation/navigation_test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    },
+    afterEach: function () {
+        $('#qunit-fixture').empty();
+    }
+});
+
+QUnit.test('Preserved at smaller screen resolutions', function (assert) {
+    var evaluateAtWidth = function (resolution) {
+        // Set the width
+        F('bitballs-navigation', 0).then(function () {
```
Comment (akagomez) [inline public/components/navigation/navigation_test.js]: Investigate `F.open()`

File: public/components/navigation/navigation_test.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+import $ from 'jquery';
+import QUnit from 'steal-qunit';
+import F from 'funcunit';
+
+F.attach(QUnit);
+
+QUnit.module('Navigation Layout', {
+    beforeEach: function () {
+        $('#qunit-fixture').html(
+            '<iframe id="navigation-iframe" src="navigation.html" />');
```
Comment (akagomez) [inline public/components/navigation/navigation_test.js]: Attempt to use the iframe for the sole purpose of the media query manipulation. Reuse everything else from the test window. 

```
<iframe><bitballs-navigation></bitballs-navigation><iframe>
```

File: public/test/utils.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import $ from 'jquery';
+import F from 'funcunit';
+
+export default {
+    insertAndPopulateIframe: function (iframeParentSelector, frag) {
```
Comment (akagomez) [inline public/test/utils.js]: Hardcode parent selector.

**Pull request merged**